README.md

```diff
@@ -1,20 +1,29 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
+
+
+** I did google for for clarify on questions 1 and 2.
+**Refereneced notes for the last question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+  A click handler is a function associated with the click event which will be called after the event occurs.
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+  I assume after the document is ready to load.
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+ The jQuery library isn't referenced.
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+  You're javascript calls, which should run after the documents are loaded.
```

Comment:
Yes, more specifically, any click handlers that need to be attached to existing HTML elements. Function definitions can go outside of `$(document).ready()`.